stackmon/status-dashboard
# feat(db): switch to sqlalchemy declarative method

declarative DB definition is a recommended way to use SQLalchemy. It
gives a slightly improved way to have a comple queries and influence how
relationships are being loaded.

- convert model to use declarative style
- adapt code in all places that require model
- add unittests for model
- add unittests for api
- add basic unittests for web (just ensure it renders something)
- merge both auth definitions to the single file
